Orckestra.Widget.FilteredSelector/Orckestra.Widget.FilteredSelector/Constants.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace Orckestra.Widget.FilteredSelector
+{
+    public static class Constants
+    {
+        public const string pageIdParamName = "PageId";

```

Comment:
Pascal casing for public properties and constants for C#

Reply:
Updated